### PR TITLE
Scheduler - add label for edit appointment

### PR DESCRIPTION
**Pull request by pomahtri**

### CODE REVIEW

File: e2e/testcafe-devextreme/tests/scheduler/a11y/scheduler.ts [hunk 1]
```diff
@@ -45,3 +52,25 @@ test('Scheduler table elements have right aria attributes', async (t) => {
     currentView: 'month',
   });
 });
+
```
Comment (Raushen) [inline e2e/testcafe-devextreme/tests/scheduler/a11y/scheduler.ts:56]: Could you please add all themes?

File: packages/devextreme/js/localization/messages/ja.json [hunk 2]
```diff
@@ -250,6 +250,7 @@
         "dxPivotGrid-filterFieldArea": "フィルター フィールドをこちらへドラッグ＆ドロップ",
 
         "dxScheduler-ariaLabel": "Scheduler. {0} view",
+
```
Comment (Raushen) [inline packages/devextreme/js/localization/messages/ja.json:253]: I'd suggest move new string here instead of blank row

Comment (pomahtri) [reply packages/devextreme/js/localization/messages/ja.json:253]: This new line appears only in generated file. In normal file (en.json), there's no blank rows

File: e2e/testcafe-devextreme/tests/scheduler/a11y/scheduler.ts (hunk 1 above)
Comment (pomahtri) [reply e2e/testcafe-devextreme/tests/scheduler/a11y/scheduler.ts:56]: Contrast is disabled. Is there any need to check other themes?

Comment (Raushen) [reply e2e/testcafe-devextreme/tests/scheduler/a11y/scheduler.ts:56]: Yes. We have different predefined settings for themes so some of them can be course of issues.

File: packages/devextreme/js/localization/messages/ja.json (hunk 2 above)
Comment (Raushen) [reply packages/devextreme/js/localization/messages/ja.json:253]: Ok

**Review by Raushen — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**